mainnet/CAKE/logo.svg

```diff
@@ -0,0 +1,14 @@
+<svg width="96" height="96" viewBox="0 0 96 96" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Change it to:
```suggestion
<svg width="256" height="256" viewBox="0 0 96 96" fill="none" xmlns="http://www.w3.org/2000/svg">
```
so CI checks can pass. Token logo size should be at least 200x200.